examples/CMakeLists.txt

```diff
@@ -45,7 +46,8 @@ if (BUILD_TESTING)
     foreach (fname ${googleapis_functions_framework_examples_unit_tests})
         string(REPLACE "/" "_" target "${fname}")
         string(REPLACE ".cc" "" target "${target}")
-        add_executable("${target}" ${fname})
+        add_executable("${target}"
+                       ${fname} site/hello_world_pubsub/hello_world_pubsub.cc)
```

Comment:
If another element was added to `googleapis_functions_framework_examples_unit_tests`, would we still want to link it with `hello_world_pubsub.cc`?

Reply:
Ooops, I selected too many targets to add these file to. Fixed.